lib/webkit_inspection_protocol.dart

```diff
@@ -7,9 +7,12 @@
 library wip;
 
 import 'dart:async';
+import 'dart:collection';
 import 'dart:convert' show JSON, UTF8;
 import 'dart:io';
 
+part 'src/dom.dart';
```

Comment:
I strongly prefer parts. This file is already almost 600 lines long, and adding 350 more lines to it will make it even harder to find things here.

In the next PR I plan on sending you, I actually split each of the domains into its own file specifically to make it easier to find the related parts of the code.